docker-compose.yml

```diff
@@ -19,19 +19,32 @@ services:
       - ./volume:/volume
     ports:
       - "8080:8080"
+    # starting with blindly following example, this flag has pretty big security impls  
+    privileged: true  
     depends_on:
       - katana
-    entrypoint: /bin/bash -c "/app/scripts/torii.sh > /volume/torii.logs"
+    environment:
+      IS_PRIMARY: "TRUE"
+      WORLD_DB: "/litefs/world.db"
+      ENTRY_CMD: "/bin/bash -c \"/app/scripts/torii.sh > /volume/torii.logs\""
```

Comment:
whereas before this was the entrypoint, we now use `litefs mount` as the entrypoint for both the torii and lore machine images, with each configuring its on `ENTRY_CMD` that the litefs will call *after* its system has started.